Chapter2

MuraokaNaoki opened the pull request.

MuraokaNaoki committed:
Impl11

MuraokaNaoki committed:
Impl11

MuraokaNaoki committed:
Impl12

MuraokaNaoki committed:
Impl13

MuraokaNaoki committed:
Impl14

MuraokaNaoki committed:
Impl15

MuraokaNaoki committed:
Impl17

MuraokaNaoki committed:
Impl18

MuraokaNaoki committed:
Impl19

7riatsu commented on 14.py:
```diff
@@ -0,0 +1,8 @@
+def extract(N):
+    return df.head(N)
+
+print(extract(4))
```
細かいけど問題文中に
`自然数Nをコマンドライン引数などの手段で受け取り`
って書いてあるから、`input()`とか利用してCLIから数字入力する形にした方が良いかも
こういうの標準入力っていうんだけど、知っておくと便利だよ
コーディングテストとか受ける場合は標準入力わからないと結構詰むw

7riatsu replied:
これ修正おねしゃす

7riatsu commented on 15.py:
```diff
@@ -0,0 +1,8 @@
+def extract(N):
+    return df.tail(N)
+
+print(extract(4))
```
ここも同様

7riatsu reviewed (commented):
全体的にできていると思いますー
pandas使うとやっぱり簡単に実装できて便利だね
あと`11.py`は？w

MuraokaNaoki committed:
８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。８行目の修正

数字をinputで受け取れるようにした。
